### PR TITLE
Implement dropout choice

### PR DESCRIPTION
**Pull request by 14thibea**
Add the `--dropout` option to the train task in order to fix the dropout hyperparameter applied to the dropout layers.

### CODE REVIEW

**Comment by mdiazmel** [comment]
Thanks @14thibea !
Everything seems ok. I'll merge this one!

**Comment by mdiazmel** [comment]
I'll fix the style issues after merge.
